src/main/resources/application.conf

```diff
@@ -122,4 +122,13 @@ akka {
     }
   }
 
+  # akka-http-caching config described here:
+  # https://doc.akka.io/docs/akka-http/current/common/caching.html#frequency-biased-lfu-cache
+  http.caching.lfu-cache {
```

Comment:
I need input on what values we would prefer to have here:

Reply:
Any thoughts, @anonymoussprocket ?

Reply:
I was thinking more an in-memory cache of the refdata requests between the database and the service, but this may work just as well.

Reply:
I was thinking about more how many entries we should store, and values for the `time-to-live` and `time-to-idle` params.